src/pages/index.tsx

```diff
@@ -1,72 +1,79 @@
-import type { NextPage } from 'next'
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import type { NextPage } from "next";
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../styles/Home.module.scss";
+import TestComponent from "../components/TestComponent";
+import Calendar from "react-calendar";
+import "react-calendar/dist/Calendar.css";
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { languages } from "../../data/expConst";
+import Select from "react-select";
 
 const Home: NextPage = () => {
+  //define our states so we can access the data the user types
+  const [value, onChange] = useState(new Date());
+  const [language, onChangeLang] = useState();
+  var [firstnameINPUT, onFirstnameCHANGE] = useState("");
+  var [lastnameINPUT, onLastnameCHANGE] = useState("");
+
+  //handler for submitting the form
+  const handleSubmit = (evt) => {
+    evt.preventDefault();
+    alert(
+      `Submitting Name ${firstnameINPUT.toString()} ${lastnameINPUT.toString()}`
+    );
+  };
+  const defaultOption = languages[0];
+
   return (
     <div className={styles.container}>
       <Head>
```

Comment:
Let's move this head into _document
https://nextjs.org/docs/advanced-features/custom-document